packages/storybook/stories/va-table.stories.jsx

```diff
@@ -52,15 +53,15 @@ const Template = args => {
         descending={descending}
       >
         <va-table-row slot="headers">
-          {columns.map(col => (
-            <span>{col}</span>
+          {columns.map((col, i) => (
+            <span key={`header-default-${i}`}>{col}</span>
           ))}
         </va-table-row>
 
-        {rows.map(row => (
-          <va-table-row>
-            {row.map(item => (
-              <span>{item}</span>
+        {rows.map((row, index) => (
+          <va-table-row key={`row-default-${index}`}>
+            {row.map((item, index2) => (
+              <span key={`row-item-default-${index2}`}>{item}</span>
```

Comment:
I added keys to mapping elements because we were getting React errors in the console in Storybook without them.